src/components/todo_action_buttons/ProgressButton.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const ProgressButton = ({setTodos, todos, todo}) =>{
+    const progressHandler = () => {
+        setTodos(
+          todos.map(item => {
+            if (item.id === todo.id) {
+              return {
+                ...item,
+                progress: !item.progress,
+              };
+            }
+            return item;
+          })
+        );
+      };
+    return(
+        <button
+        onClick={progressHandler}
+        disabled={todo.completed ? true : false}
```

Comment:
`if (true) return true`? Same as boolean comparison above, you can just use `disabled={todo.completed}`

Reply:
fixed